### PR TITLE
[PL-57598]: Doc update for GCS Streaming destination

### PR DESCRIPTION
**Pull request by GokulBansal0**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: Doc update for GCS Streaming destination
* Jira/GitHub Issue numbers (if any): [PL-57598](https://harness.atlassian.net/browse/PL-57598)
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-57598]: https://harness.atlassian.net/browse/PL-57598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 1]
```diff
@@ -52,6 +52,36 @@ To add a streaming destination:
 After you add your streaming destination, you're ready to configure the streaming connector.
 
 <Tabs>
+<TabItem value="GCS" label="GCP GCS" default>
+
+To configure the GCP GCS streaming connector:
```
Comment (pratmit) [inline docs/platform/governance/audit-trail/audit-streaming.md:57]: Is this behind a FF?

**Review by pratmit — COMMENTED** [review]
Added a comment

File: docs/platform/governance/audit-trail/audit-streaming.md (hunk 1 above)
Comment (GokulBansal0) [reply docs/platform/governance/audit-trail/audit-streaming.md:57]: I am waiting for confirmation about that.

Comment (GokulBansal0) [reply docs/platform/governance/audit-trail/audit-streaming.md:63]: Added the ff

**Review by pratmit — APPROVED** [review]

**Pull request merged**